org.mixedrealitytoolkit.input/Assets/Prefabs/Experimental-XRI3/CopyNewPrefabsForDevelopment.ps1

```diff
@@ -0,0 +1,5 @@
+﻿Copy-Item '.\MRTK XRI Gaze.prefab' '..\MRTK Gaze Controller.prefab'
```

Comment:
Why is this needed? This copy is likely a onetime operation needed only be you.

Please don't include, so to avoid extra maintenance cost.

Reply:
Reverted with https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/743/commits/ae962a4080fa69b164976601408ca163bb603ee0 , resolving.